### DISCUSSION TITLE
MadmeasureHDR Optimizer out

### DISCUSSION THREAD

**Discussion by BadSector81** [quietvoid/dovi_tool#59] [Ideas]
Hello! First of all I thank you your hard work on dovi_tool.
I would like to ask that are you planning to make this too to be compatible with MadmeasureHDR Optimizer's  measurement files? I read that it can create more precise measurements. 

( [https://dereferer.me/?https://www.avsforum.com/threads/madvr-tool-madmeasurehdr-optimizer-measurements-dynamic-clipping-target-nits.3040072/](url)

Thank you!
BadSector81

**Comment by quietvoid** [comment]
Yes it is planned.

**Reply by BadSector81** [reply]
Great news! Thanks!

**Comment by quietvoid** [comment]
@BadSector81 I'm unable to use the optimizer with any of my measurement files.
Do you have some samples?

The ones I have are using `flags=3`, and I can't find documentation about it.
madshi only mentioned a use for `flags=2`

Nevermind, I found the reason: https://www.avsforum.com/threads/madvr-tool-madmeasurehdr-optimizer-measurements-dynamic-clipping-target-nits.3040072/page-16#post-57431312

**Comment by quietvoid** [comment]
This has been implemented in 1.1.3.
The target nits are used as L1 peak nits per frame, but they are not as smooth as per-shot metadata.
The scene cuts and average nits are dynamic per-shot.

**Reply by BadSector81** [reply]
Thank you!!
